_posts/2017/07/2017-07-05-google-assistant-96boards-instructions.md

```diff
@@ -74,7 +74,7 @@ tags:
 ---
 # **Introduction**
 
-**This blog is subject to going out of date. Please** [visit this project](https://github.com/96boards/projects/tree/master/google_assistant) **in the** [96Boards projects repository](https://github.com/96boards/projects) **for the most up to date instruction set.**
+**This blog is subject to going out of date. Please** [visit this project](https://github.com/96boards-projects/google_assistant) **in the** [96Boards projects GitHub repository](https://github.com/96boards-projects) **for the most up to date instruction set.**
```

Comment:
s/96Boards projects GitHub repository/96Boards Projects Org

This is the name we should be using for pointing to new projects org.